### PR TITLE
quick-workaround: syntax error

### PR DESCRIPTION
**Pull request by sHaggYcaT**
**What this PR does / why we need it**:

Due syntax error, few our tests now broken. Please approve it fix compilation error issue

**Release note:**
```release-note
NONE
```

### CODE REVIEW

**Comment by sHaggYcaT** [comment]
#621 is more correct fix

**Comment by sHaggYcaT** [comment]
Looks like #621 still not merged :( Can we merge this patch, then Yan bring correct fix?

File: tests/network_migration_test.go [hunk 1]
```diff
@@ -150,11 +161,14 @@ var _ = Describe("[rfe_id:1150][crit:high][vendor:cnv-qe@redhat.com][level:compo
...
+
+*/
 func curlReq(ip string, port string, vmi *v1.VirtualMachineInstance, resp string) {
-        ktests.WaitUntilVMIReady(vmi, ktests.LoggedInCirrosExpecter)
-        err := ktests.CheckForTextExpecter(vmi, []expect.Batcher{
-                &expect.BSnd{S: fmt.Sprintf("curl --silent --connect-timeout 5 --head %s%s  | grep 'HTTP/1.1 200 OK' | wc -l \n", ip, port)},
-                &expect.BExp{R: resp},
-        }, 60)
-        Expect(err).ToNot(HaveOccurred())
+	ktests.WaitUntilVMIReady(vmi, ktests.LoggedInCirrosExpecter)
```
Comment (MarSik) [inline tests/network_migration_test.go]: Tabs vs. spaces.

**Comment by MarSik** [comment]
Yeah, I do not know what is worse.. the original breakage should have never happened. Lets merge this one to unblock our work.

**Pull request merged**